### PR TITLE
CFEE 1650

### PR DESCRIPTION
**Pull request by manojtyagi2021**
Create plugin skeleton code that prints "Hello World" on the console
## Description

- Create plugin skeleton code that prints "Hello World" on the console
- Set up GitHub workflow which will trigger on pull Request
- Compile the plugin code for multiple platforms
- Push binaries to the Current Release Tag using GitHub workflow

## Motivation and Context

Creation of a CF CLI Plugin to establish a container-to-container (app to app) network constraint
[Issue Link](https://hclcloud.atlassian.net/browse/CFEE-1628)

## Screenshots (if appropriate)
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code has security implications.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+
+```shell script
+# Clone repo and cd into it then...
+docker build --no-cache --tag hclcnlabs/cfmr-cf-cli-plugin-network-policy:1.0.0 .
+```
+
+```shell script
+# Push the image to dockerhub(hclcnlabs)...
+docker push hclcnlabs/cfmr-cf-cli-plugin-network-policy:1.0.0
+```
```
Comment (jbuns) [inline README.md]: Is this still true?

**Review by jbuns — COMMENTED** [review]
Comments

File: README.md (hunk 1 above)
Comment (manojtyagi2021) [reply README.md]: This is not relevant now so have removed it.
I will come back to Readme later when the plugin functionality will be completed.

**Review by jbuns — APPROVED** [review]
lgtm

**Pull request merged**